spec/mailers/request_mailer_spec.rb

```diff
@@ -16,15 +16,10 @@
       receive_incoming_mail('incoming-request-plain.email', ir.incoming_email)
       expect(ir.incoming_messages.size).to eq(2) # one more arrives
       expect(ir.info_request_events[-1].incoming_message_id).not_to be_nil
-
-      deliveries = ActionMailer::Base.deliveries
-      expect(deliveries.size).to eq(1)
-      mail = deliveries[0]
-      expect(mail.to).to eq([ 'bob@localhost' ]) # to the user who sent fancy_dog_request
-      deliveries.clear
```

Comment:
In the cases where I've removed these, I think the only mail being cleared was the new response notification I've now removed. I'll double check though.